buildpack/bp_descriptor.go

```diff
@@ -24,9 +24,10 @@ type StackMetadata struct {
 }
 
 type TargetPartial struct {
-	OS          string `json:"os" toml:"os"`
 	Arch        string `json:"arch" toml:"arch"`
 	ArchVariant string `json:"arch-variant" toml:"arch-variant"`
+	ID          string `json:"id" toml:"id"`
```

Comment:
```suggestion
```
Actually this isn't part of the buildpack spec - we provide the run image's target id in `CNB_TARGET_ID` but it isn't part of buildpack.toml for the purposes of determining which run images the buildpack is compatible with.

Reply:
💡